apps/common/behaviours/generate-pdf.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const pdf = require('html-pdf');
+const UploadModel = require('../../common/models/file-upload');
+const concat = require('concat-stream');
+const path = require('path');
+
+module.exports = superclass => class extends superclass {
+  saveValues(req, res, callback) {
+    super.saveValues(req, res, (err, values) => {
```

Comment:
In fact, the more I think about this, the more I think it should be `process` and not `saveValues` for exactly that reason. 

`process` is generally where we write to `req.form.values`